tasks/copy_to_glacier/schemas/config.json

```diff
@@ -5,7 +5,7 @@
   "description": "The config for the copy_to_glacier Lambda.",
   "type": "object",
   "properties": {
-    "excludeFileTypes": {
+    "OrcaExcludedFileRegEx": {
       "type": ["array", "null"],
       "description": "A list of file extensions to ignore when copying files.",
```

Comment:
While we're at it, want to update this as well? "A list of file extensions in the form of regular expressions..."?

Reply:
updated to `A list of file extensions to ignore when copying files in the form of regular expressions '^.*{file_type}$'`

Reply:
Sorry, was fooled by the new name. These aren't actually regexes.